elements/lisk-utils/test/math/q.spec.ts

```diff
@@ -0,0 +1,325 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { Q_OPERATION, q, q96 } from '../../src/math';
+
+/**
+ * For the test cases `binary` represents the calculation result in binary.
```

Comment:
Is `binary` value ever used as an input or output of any test, or is it there only to support some manual checks?

It looks like we could get rid of it completely.

Reply:
binary is used more like test description.
From the binary result, the exact q value is constructed with 8bit accuracy

Reply:
Cleaned up and pushed